README.md

```diff
@@ -31,63 +39,84 @@ The Polaris dashboard is a way to get a simple visual overview of the current st
 
 Our default standards in Polaris are rather high, so don’t be surprised if your score is lower than you might expect. A key goal for Polaris was to set a high standard and aim for great configuration by default. If the defaults we’ve included are too strict, it’s easy to adjust the configuration as part of the deployment configuration to better suit your workloads.
 
-### Deploying
+## Webhook
+
+Polaris includes experimental support for an optional validating webhook. This accepts the same configuration as the dashboard, and can run the same validations. This webhook will reject any deployments that trigger a validation error. This is indicative of the greater goal of Polaris, not just to encourage better configuration through dashboard visibility, but to actually enforce it with this webhook. *Although we are working towards greater stability and better test coverage, we do not currently consider this webhook component production ready.*
+
+Unfortunately we have not found a way to display warnings as part of `kubectl` output unless we are rejecting a deployment altogether. That means that any checks with a severity of `warning` will still pass webhook validation, and the only evidence of that warning will either be in the Polaris dashboard or the Polaris webhook logs.
 
-To deploy Polaris with kubectl:
+## Installation and Usage
+Polaris can be installed on your cluster using kubectl or Helm. It can also
+be run as a local binary, which will use your kubeconfig to connect to the cluster
+or run against local YAML files.
 
+### kubectl
+#### Dashboard
 ```
 kubectl apply -f https://raw.githubusercontent.com/reactiveops/polaris/master/deploy/dashboard.yaml
+kubectl port-forward --namespace polaris svc/polaris-dashboard 8080:80
 ```
-
-Polaris can also be deployed with Helm:
-
+#### Webhook
 ```
-helm upgrade --install polaris deploy/helm/polaris/ --namespace polaris
+kubectl apply -f https://raw.githubusercontent.com/reactiveops/polaris/master/deploy/webhook.yaml
```

Comment:
Looks like some of these are due to a squash merge - I will see if I can rebase to resolve the visual inconsistency.